### PR TITLE
Wrong amount in Send actionsheet #2661

### LINKED ISSUE #2599: Check if CoinGecko API is good for our purpose
We already use CoinGecko for ETH prices, it looks like we might be able to use them for other tokens as well as historical prices:

https://www.coingecko.com/api/documentations/v3#/coins/get_coins__id__market_chart

I think there's enough data for #2274

### LINKED ISSUE #2661: Wrong amount in Send actionsheet
I tried to send 5 DAI, but it displays my total balance under "Amount".

### LINKED ISSUE #2662: No way to enter decimals when sending funds
Maybe only for Europeans?

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes #2662, Closes  #2661

### CODE REVIEW

**Comment by hboon** [comment]
So this PR would fix #2655 too?

File: AlphaWallet/Settings/Types/Config.swift [hunk 1]
```diff
@@ -36,7 +36,7 @@ struct Config {
     }
 
     static var locale: Locale {
```
Comment (hboon) [inline AlphaWallet/Settings/Types/Config.swift:38]: I know you mentioned this in #https://github.com/AlphaWallet/alpha-wallet-ios/issues/2655#issuecomment-803947279, but I don't understand.

When do we use `Config.locale` and when do we use `Config.getLocale()`? In other words, why do we want `Config.locale` to return `Locale.current` when the user has overridden the language in in-app Settings > Advanced > Change Language?

**Review by hboon — CHANGES_REQUESTED** [review]
I see this when iOS region is Singapore+English and in-app setting is Polish, after app is restarted. Notice it shows "0,1" in the amount textfield, but "0.1" in the actionsheet.

Screenshot 2021-03-23 at 1 00 22 PM

**Comment by hboon** [comment]
@vladyslav-iosdev also see if https://github.com/AlphaWallet/alpha-wallet-ios/issues/2662#issuecomment-804614729 and the comments after that might be relevant?

**Comment by hboon** [comment]
@all-contributors please add @vladyslav-iosdev for code

**Comment by vladyslav-iosdev** [comment]
> #2655

yep, it should

**Comment by hboon** [comment]
@vladyslav-iosdev it works on iOS 12.4 simulator, but I tried on an iPhone 11 Pro (device) with iOS 14.5 and changing the locale in-app to and after restarting the app, the keyboard and textfields still show "." as the decimal separator. Does it work for you?

**Comment by hboon** [comment]
@vladyslav-iosdev going to merge this in and I'll create a separate PR to hide (and undo) the in-app language settings for now until we sort it out.

**Review by hboon — APPROVED** [review]

**Pull request merged**